maven-core/src/main/java/org/apache/maven/DefaultMaven.java

```diff
@@ -466,7 +468,9 @@ private void validatePrerequisitesForNonMavenPluginProjects( List<MavenProject>
         }
     }
 
-    private void validateActivatedProfiles( List<MavenProject> projects, List<String> activeProfileIds )
+    private void validateActivatedProfiles( List<MavenProject> projects,
+                                            List<String> activeProfileIds,
+                                            List<String> inactiveProfiles )
```

Comment:
Let's name this inactiveProfileIds for consistency

Reply:
I followed the name of method used in the call. I agree that local naming style should be preserved here - will update that.

Reply:
This has been implemented.